tests_integration/mock_server_addon.py

```diff
@@ -0,0 +1,60 @@
+"""
```

Comment:
Maybe add a shebang too?
```suggestion
#!/usr/bin/env mitmdump -s
"""
```